src/urllib3/response.py

```diff
@@ -772,14 +802,14 @@ def _update_chunk_length(self):
             self.close()
             raise InvalidChunkLength(self, line)
 
-    def _handle_chunk(self, amt):
+    def _handle_chunk(self, amt: Optional[int]) -> bytes:
         returned_chunk = None
         if amt is None:
             chunk = self._fp._safe_read(self.chunk_left)
             returned_chunk = chunk
             self._fp._safe_read(2)  # Toss the CRLF at the end of the chunk.
             self.chunk_left = None
-        elif amt < self.chunk_left:
+        elif self.chunk_left is not None and amt < self.chunk_left:
```

Comment:
I am not sure but I think it can be `None`. because based on the code [_update_chunk_length](https://github.com/urllib3/urllib3/blob/dfd8b580adb4f87388310c11b5c76d989bd6c4e3/src/urllib3/response.py#L761) will be called before [_handle_chunk](https://github.com/urllib3/urllib3/blob/dfd8b580adb4f87388310c11b5c76d989bd6c4e3/src/urllib3/response.py#L775), and there is:
```
        if self.chunk_left is not None:
            return
```
So, it seems it can be `None`.